problemsets-mat223/modules/module13-exercises.tex

```diff
@@ -1,17 +1,37 @@
 \begin{exercises}
 	\begin{problist}
-		\prob  Example problem with parts
-		\begin{enumerate}
-			\item   The first part
-			\item   The second part
-		\end{enumerate}
-		\begin{solution}
-		Solutions to the parts
-			\begin{enumerate}
-				\item   $(2,4,8)$
-				\item   $(15,6,-21)$
-			\end{enumerate}
-		\end{solution}
-
+		\prob Let $\mathcal A=\Set{\vec a_1,\vec a_2}$ where $\vec a_1=\mat{2\\1}_{\mathcal E}$ and $\vec a_2=\mat{1\\-2}_{\mathcal E}$ and let $\mathcal B=\Set{\vec b_1,\vec b_2}$ where $\vec b_1=\mat{3\\-1}_{\mathcal E}$ and $\vec b_2=\mat{-2\\3}_{\mathcal E}$ be bases for $\R^2$. 
+	    \begin{enumerate}
+	        \item Given that $[\vec x]_{\mathcal A}=\mat{1\\-1}$, find $[\vec x]_{\mathcal B}$.
+	        \item Find the change of basis matrix $\BasisChange{\mathcal A}{\mathcal B}$.
+	    \end{enumerate}
+	    
+	    \prob Let $\mathcal A=\Set{\vec a_1,\vec a_2,\vec a_3}$ where $\vec a_1=\mat{2\\1\\0}_{\mathcal E}$, $\vec a_2=\mat{1\\-2\\0}_{\mathcal E}$, and $\vec a_3=\mat{0\\0\\1}_{\mathcal E}$.
+	    
+	    Let $\mathcal B=\Set{\vec b_1,\vec b_2,\vec b_3}$ where $\vec b_1=\mat{3\\0\\-1}_{\mathcal E}$, $\vec b_2=\mat{-2\\0\\3}_{\mathcal E}$, and $\vec b_3=\mat{0\\1\\0}_{\mathcal E}$.
+	    
+	    Then  both $\mathcal A$ and $\mathcal B$ are bases for $\R^3$. 
+	    \begin{enumerate}
+	        \item Given that $[\vec x]_{\mathcal A}=\mat{1\\1\\1}$, find $[\vec x]_{\mathcal B}$.
+	        \item Find the change of basis matrix $\BasisChange{\mathcal A}{\mathcal B}$.
+	    \end{enumerate}
+	    
+	    \prob Let $\mathcal B=\Set{\vec b_1,\vec b_2}$ where $\vec b_1=\mat{1\\1}_{\mathcal E}$ and $\vec b_2=\mat{-1\\1}_{\mathcal E}$ be a basis for $\R^2$.
+	    
+	    For each $\mathcal T:\R^2\to\R^2$ defined below, compute $[\mathcal T]_{\mathcal E}$ and $[\mathcal T]_{\mathcal B}$.
+	        \begin{enumerate}
+	            \item   $\mathcal T$ is the linear transformation that doubles every vector.
```

Comment:
You don't need to say `linear` each time. It's redundant. There is only one transformation that doubles every vector, and it is a linear transformation.